index.bs

```diff
@@ -74,903 +60,149 @@ specification uses that specification and terminology.
 
-<pre class="idl">
-[Constructor(DOMString host, unsigned short port), Exposed=Window]
-interface QuicTransport : QuicTransportBase {
-};
-</pre>
+# Interface Mixin `GenericTransformStream` #  {#generic-transform-stream}
 
-When the {{QuicTransport/QuicTransport()}} constructor is invoked, the user
-agent MUST run the following steps:
-1. If {{QuicTransport/QuicTransport(host, port)/port}} is 0, [=throw=] an
-   {{NotSupportedError}} and abort these steps.
-1. Let |transport| be a newly constructed {{QuicTransport}} object.
-1. Let |transport| have a
-   <dfn attribute for="QuicTransport">\[[OutgoingStreams]]</dfn> internal slot
-   representing a sequence of {{OutgoingStream}} objects, initialized to empty.
-1. Let |transport| have a
-   <dfn attribute for="QuicTransport">\[[ReceivedStreams]]</dfn> internal slot
-   representing a {{ReadableStream}} of {{IncomingStream}} objects, initialized
-   to empty.
-1. Let |transport| have a
-  <dfn attribute for="QuicTransport">\[[ReceivedBidirectionalStreams]]</dfn>
-  internal slot representing a {{ReadableStream}} of {{IncomingStream}}
-  objects, initialized to empty.
-1. Let |transport| have a
-   <dfn attribute for="QuicTransport">\[[WebTransportState]]</dfn> internal
-   slot, initialized to `"connecting"`.
-1. Let |transport| have a
-   <dfn attribute for="QuicTransport">\[[SentDatagrams]]</dfn> internal slot
-   representing a {{WritableStream}} of {{Uint8Array}}s, initialized to empty.
-1. Let |transport| have a
-   <dfn attribute for="QuicTransport">\[[ReceivedDatagrams]]</dfn> internal
-   slot representing a {{ReadableStream}} of {{Uint8Array}}s, initialized to
-   empty.
-1. Run these steps in parallel:
-    1. Let |clientOrigin| be |transport|'s [=relevant settings object=]'s
-       [=origin=],
-       [serialized](https://html.spec.whatwg.org/multipage/origin.html#ascii-serialisation-of-an-origin).
-    1. Establish a QUIC connection to the address identified by the given
-       {{QuicTransport/QuicTransport(host, port)/host}} and
-       {{QuicTransport/QuicTransport(host, port)/port}} following the
-       procedures in [[!WEB-TRANSPORT-QUIC]] section 3 and using |clientOrigin|
-       as the "origin of the client" referenced in section 3.2. 
-    1. If the connection fails, set |transport|'s {{[[WebTransportState]]}}
-       internal slot to `"failed"` and abort these steps.
-    1. Set |transport|'s {{[[WebTransportState]]}} internal slot to
-       `"connected"`.
-1. Return |transport|.
-
-# Interface Mixin `OutgoingStream` #  {#outgoing-stream}
-
-An <dfn interface>OutgoingStream</dfn> is a stream that can be written to, as
-either a {{SendStream}} or a {{BidirectionalStream}}.
+The <dfn interface>GenericTransformStream</dfn> interface mixin represents the concept of a transform stream in IDL. It is not a TransformStream, though it has the same interface and it delegates to one.
```

Comment:
I think you need to write it like `{{GenericTransformStream}}` so that it links to the definition in the IDL.